core/jobs/src/test/java/com/cognifide/aet/job/common/comparators/source/SourceComparatorRegExpTest.java

```diff
@@ -26,12 +26,12 @@
 @RunWith(Theories.class)
 public class SourceComparatorRegExpTest {
 
-  private SourceComparator tested;
+  private CodeFormatter tested;
 
   @Before
   public void setUp() {
     // initialized with nulls as tested method is stateless
-    tested = new SourceComparator(null, null, null, null);
+    tested = new CodeFormatter();
   }
 
   @DataPoints
```

Comment:
Could you please add few lines with whitespace charactes (like `\t`, ` `) for sample data?

Reply:
done